### PR TITLE
Add Datablend

### PR DESCRIPTION
**Pull request by bmp02050**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [ x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hi @bmp02050 thank you so much for your submission. In this pull request there are other connectors files. can you please exclude other connectors files and submit again? Thank you very much.

File: certified-connectors/Datablend/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,324 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Datablend",
+    "description": "Connect to the Datablend API for a cost-effective iPaaS solution supporting finance and accounting ",
+    "version": "1.0",
+	"contact":{
+		"name":"Datablend Support",
```
Comment (sriyen-msft) [inline certified-connectors/Datablend/apiDefinition.swagger.json:8]: @bmp02050 Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: certified-connectors/ShareEffect/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1,402 +0,0 @@
-{
```
Comment (sriyen-msft) [inline certified-connectors/ShareEffect/apiDefinition.swagger.json:1]: @bmp02050 Please let us know the reason for deleting another connector files in your PR. It is not allowed to delete a connector for any partner..

File: certified-connectors/Datablend/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "h0n3r0rjg0adkoldvu6ulv1br",
```
Comment (sriyen-msft) [inline certified-connectors/Datablend/apiProperties.json:8]: @bmp02050 Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
`"clientId": "[[DUMMY]]"`

File: certified-connectors/Datablend/readme.MD [hunk 4]
```diff
@@ -0,0 +1,17 @@
+## Datablend API Connector
+Datablend is a powerful iPaaS solution with a focus is serving the needs of CFOs, controllers, and financial planning and analysis (budgeting) professionals as they look to streamline processes, speed up closes, and increase confidence in reporting.
+
```
Comment (sriyen-msft) [inline certified-connectors/Datablend/readme.MD:3]: @bmp02050 Please refer the templates and add required sections, for example Publisher section is missing.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/ShareEffect/apiDefinition.swagger.json (hunk 2 above)
Comment (bmp02050) [reply certified-connectors/ShareEffect/apiDefinition.swagger.json:1]: I didn't mean to delete them. I was removing them from the commit.

File: certified-connectors/Datablend/readme.MD (hunk 4 above)
Comment (bmp02050) [reply certified-connectors/Datablend/readme.MD:3]: This is completed

File: certified-connectors/Datablend/apiProperties.json (hunk 3 above)
Comment (bmp02050) [reply certified-connectors/Datablend/apiProperties.json]: Completed

File: certified-connectors/Datablend/apiDefinition.swagger.json (hunk 1 above)
Comment (bmp02050) [reply certified-connectors/Datablend/apiDefinition.swagger.json:8]: Changed Indentation to 2 space tab

File: certified-connectors/Datablend/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/groups/search": {
+      "post": {
+        "responses": {
+          "default": {
```
[the remark this answers is not in the document]
Comment (bmp02050) [reply certified-connectors/Datablend/apiDefinition.swagger.json]: Updated

File: certified-connectors/Datablend/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/workflows/search": {
+      "post": {
+        "responses": {
+          "default": {
```
[the remark this answers is not in the document]
Comment (bmp02050) [reply certified-connectors/Datablend/apiDefinition.swagger.json]: Updated

File: certified-connectors/Datablend/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,324 @@
...
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/workflow-executions": {
+      "post": {
+        "responses": {
+          "default": {
```
[the remark this answers is not in the document]
Comment (bmp02050) [reply certified-connectors/Datablend/apiDefinition.swagger.json]: Updated

File: certified-connectors/Datablend/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,324 @@
...
+			"description": "Get Query Execution By Id"
+          }
+        ]
+      }
+    },
+    "/workflow-executions/search": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```
[the remark this answers is not in the document]
Comment (bmp02050) [reply certified-connectors/Datablend/apiDefinition.swagger.json:110]: Updated

File: certified-connectors/Datablend/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,324 @@
...
+            }
+          }
+        ]
+      }
+    },
+    "/query-executions/{id}/results": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```
[the remark this answers is not in the document]
Comment (bmp02050) [reply certified-connectors/Datablend/apiDefinition.swagger.json:84]: Updated

File: certified-connectors/ShareEffect/apiDefinition.swagger.json (hunk 2 above)
Comment (bmp02050) [reply certified-connectors/ShareEffect/apiDefinition.swagger.json:1]: I was also told to exclude them and try again

File: certified-connectors/Datablend/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/Datablend/apiDefinition.swagger.json:8]: @bmp02050 Can you check the Indentation (property alignment) at line number 27 and fix similar alignment issues of all the properties in the swagger.

File: certified-connectors/Datablend/apiDefinition.swagger.json [hunk 10]
```diff
@@ -0,0 +1,959 @@
...
+            }
+          }
+        ]
+      }
+    },
+    "/workflows/search": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
```
Comment (sriyen-msft) [inline certified-connectors/Datablend/apiDefinition.swagger.json:227]: @bmp02050 Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

File: certified-connectors/Datablend/readme.MD (hunk 4 above)
Comment (sriyen-msft) [reply certified-connectors/Datablend/readme.MD:3]: @bmp02050 The publisher name section is missing. , [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) is the template you can refer to update your connector's readme.md file.

File: certified-connectors/ShareEffect/apiDefinition.swagger.json (hunk 2 above)
Comment (sriyen-msft) [reply certified-connectors/ShareEffect/apiDefinition.swagger.json:1]: @bmp02050 Seems you need to update your fork with 'dev' repo. If we approve and merge your PR, the "ShareEffect" connector would be removed from the repo.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Datablend/apiDefinition.swagger.json (hunk 1 above)
Comment (bmp02050) [reply certified-connectors/Datablend/apiDefinition.swagger.json]: Updated

File: certified-connectors/ShareEffect/apiDefinition.swagger.json (hunk 2 above)
Comment (bmp02050) [reply certified-connectors/ShareEffect/apiDefinition.swagger.json]: I pulled the DEV and rebased it to my version. The ShareEffect stuff is still in there. Not sure how to exclude it without it deleting.

**Comment by bmp02050** [comment]
@sriyen-msft What else is left? What do I need to do to complete this?

**Comment by sriyen-msft** [comment]
> @sriyen-msft What else is left? What do I need to do to complete this?

@bmp02050 Please follow the [documentation](https://github.com/microsoft/PowerPlatformConnectors#creating-a-fork) and create a valid fork using our "dev" branch. And you can add your connector files to create a new PR. We did not hear or not having any issues with the dev branch for any other partner. Please let me know if you need more assistance, so that we can schedule a meeting and help you to raise a valid PR. Thanks

**Comment by bmp02050** [comment]
Rebuilding this and trying a new pull request

**Pull request closed without merging**